Formula/lilypond.rb

```diff
@@ -36,14 +45,17 @@ class Lilypond < Formula
   depends_on "python@3.9"
 
   uses_from_macos "flex" => :build
+  uses_from_macos "perl" => :build
+  uses_from_macos "rsync" => :build
+  uses_from_macos "zip" => :build
 
   def install
     system "./configure",
             "--prefix=#{prefix}",
             "--datadir=#{share}",
             "--with-texgyre-dir=#{Formula["texlive"].opt_share}/texmf-dist/fonts/opentype/public/tex-gyre",
             "--disable-documentation"
-    ENV.prepend_path "LTDL_LIBRARY_PATH", Formula["guile@2"].lib
+    ENV.prepend_path "LTDL_LIBRARY_PATH", Formula["guile@2"].opt_lib
```

Comment:
These two are actually the same due to some magic in `Formula#lib`. But this change is stylistically consistent with what we do elsewhere, so this is fine.